### PR TITLE
Update SRG-OS-000342-GPOS-00133 for RHEL 9 STIG

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:

Add missing SRG requirements, fixtexts, update OCIL and OCIL clauses.

#### Rationale:

RHEL 9 STIG

### CODE REVIEW

**Review by Mab879 — CHANGES_REQUESTED** [review]

File: controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ controls:
             - auditd_audispd_disk_full_action
```
[the remark this answers is not in the document]
Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: But disk_full_action setting in /etc/audit/auditd.conf is a differrent thing than the disk_full_action setting in /etc/audit/audisp-remote.conf. The audisp-remote plugin is a plugin that performs remote logging to an aggregating remote server. The  disk_full_action setting in /etc/audit/audisp-remote.conf configures the action that should be taken when the remote server is out of disk space. OTOH the disk_full_action setting in /etc/audit/auditd.conf sets the action that should be taken when the local system is out of disk space.

[the remark this answers is not in the document]
Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: And this SRG says "... must off-load audit records onto a different system ..." so I think that the /etc/audit/audisp-remote.conf which handles the remote system fits into here.

[the remark this answers is not in the document]
Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: I'm not finding "audisp-remote.conf" anywhere in the RHEL8 STIG, but they are in the RHEL7 STIG. Was rsyslog used in its place? Was it a mistake that the audisp rules were removed from the RHEL8 STIG?

[the remark this answers is not in the document]
Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: I have no idea.
@ggbecker Do you know something about this?

[the remark this answers is not in the document]
Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: I have found that the rule has never been a part of RHEL 8 STIG so it hasn't been removed so I can't find why it isn't there :smile: 

I have found that some of these rules have been a part of RHEL 8 OSPP but have been removed from RHEL 8 OSPP due to CC certification. According to https://bugzilla.redhat.com/show_bug.cgi?id=2068563 it's been removed in PR https://github.com/ComplianceAsCode/content/pull/8508. But the justification in that BZ isn't a justification that we could reuse for STIG:

> When reducing the scope of the certification, this new never-before-certified feature we originally planned for 8.4+ didn't make the cut.

The rule auditd_audispd_disk_full_action is a part of multiple STIG profiles: RHEL 7, SLE 12, SLE 15, RHV 4, OL 7. 

So I'm inconclusive.

[the remark this answers is not in the document]
Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: It's looking like the Rsyslog was trying to be used for this purpose. https://stigs.mab879.com/products/rhel8/v1r5/RHEL-08-030710/

[the remark this answers is not in the document]
Comment (jan-cerny) [reply controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml:10]: so it seems that there is no good way to off-load audit records to a different system

**Comment by jan-cerny** [comment]
I have rebased this on the top of the latest upstream branch and I have removed the audisp related rules.

**Pull request merged**